examples/adduser.rs

```diff
@@ -14,7 +11,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     eprintln!("{:?}", admin_token);
 
-    let admin = KeycloakAdmin::new(&url, admin_token, client);
+    let mut admin = KeycloakAdmin::new(&url, admin_token, client);
```

Comment:
You need to mutate state to call any function from admin. This puts a lot of restrictions on library users, which is obviously not good.